### PR TITLE
Fixed delete (singular) overloads so that they send a valid pointer.

### PR DESCRIPTION
**Pull request by NogginBops**
### Purpose of this PR

Fixed the delete singular overloads so that they don't crash.

### Testing status

Hand tested with renderdoc and VS debugger to make sure it does the right thing.

### CODE REVIEW

**Comment by deccer** [comment]
This should fix #1277 then.

File: src/GeneratorV2/Process/Processor.cs [hunk 1]
```diff
@@ -1020,14 +1020,20 @@ public DeleteOverloadLayer(Parameter lengthParameter, Parameter inParameter, Par
                     PointerParameter = pointerParameter;
                 }
 
+                private IndentedTextWriter.Scope Scope;
                 public void WritePrologue(IndentedTextWriter writer, NameTable nameTable)
                 {
                     writer.WriteLine($"{LengthParameter.Type.ToCSString()} {nameTable[LengthParameter]} = 1;");
-                    writer.WriteLine($"{PointerParameter.Type.ToCSString()} {nameTable[PointerParameter]} = ({PointerParameter.Type.ToCSString()}){nameTable[InParameter]};");
+                    writer.WriteLine($"fixed({PointerParameter.Type.ToCSString()} {nameTable[PointerParameter]} = &{nameTable[InParameter]})");
+                    //writer.WriteLine($"{PointerParameter.Type.ToCSString()} {nameTable[PointerParameter]} = ({PointerParameter.Type.ToCSString()}){nameTable[InParameter]};");
```
Comment (deccer) [inline src/GeneratorV2/Process/Processor.cs:1028]: This line should be removed then

**Review by deccer — CHANGES_REQUESTED** [review]
One line needs to be removed

**Review by frederikja163 — APPROVED** [review]
Merging now, will go in and remove L1028 at some other point. I also have a few other changes i want to get merged in before we do another prerelease, specifically regarding the drawelements methods, with an offset.

**Pull request merged**